sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/ColumnVector.java

```diff
@@ -99,6 +185,26 @@ public void reset() {
   /**
    * Sets the value at rowId to `value`.
    */
+  public abstract void putByte(int rowId, byte value);
```

Comment:
Yea, we're still missing those two types and probably decimal. I'll do those in a follow up.